org.eclipse.lyo.oslc4j.build/pom.xml

```diff
@@ -28,6 +28,7 @@
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
     <maven.compiler.source>1.8</maven.compiler.source>
     <maven.compiler.target>1.8</maven.compiler.target>
+    <jena.version>3.6.0</jena.version>
```

Comment:
I assume we will wait with this merge into master until we migrate to 3.6.0 then.

Reply:
No, this patch *is* the migration to 3.6.0.

Am I clear to merge?

Reply:
sure